### DISCUSSION TITLE
Cannot filter with multiple fields

### DISCUSSION THREAD

**Discussion by hkvongit** [pocketbase/pocketbase#4134] [Q&A] [answered]
I am having a collection with the below schema:
<img width="468" alt="image" src="https://github.com/pocketbase/pocketbase/assets/45459914/36f39202-f5a3-4606-9d0e-9f1ddfa3f30c">
But I cannot filter with multiple parameters via API:
```md
{{POCKET_BASE_URL}}/api/collections/jobApplications/records?filter=(jobApplicationStatus='ACCEPTED' && jobInvitationStatus='DENIED')
```
it is giving me the below error
<img width="1174" alt="image" src="https://github.com/pocketbase/pocketbase/assets/45459914/bc6a3af8-7504-4e6c-b193-1330844c1dfd">

When I check the Pocket base log, it is showing bracket is missing but understand where the bracket is missing.
<img width="1356" alt="image" src="https://github.com/pocketbase/pocketbase/assets/45459914/f024bed3-8095-47c5-abab-b7ff7e450c9a">

**Comment by ganigeorgiev** [comment, marked as answer]
The `filter` query parameter value must be properly URL encoded (the SDKs do this automatically). Try with:

```
?filter=%28jobApplicationStatus%3D%27ACCEPTED%27%20%26%26%20jobInvitationStatus%3D%27DENIED%27%29
```

If that's not the case, to rule out PocketBase issue you can try to paste your rule directly in the Admin UI records listing searchbar.

**Reply by hkvongit** [reply]
Thanks a lot @ganigeorgiev, your answer solved my issue.
